requirements_dev.txt

```diff
@@ -4,5 +4,8 @@ pytest-watch==4.1.0
 coverage==4.0.3
 mock==2.0.0
 
-# PyMongo is needed for one of the unit tests:
+# PyMongo and Athena dependencies are needed for some of the unit tests:
+# (this is not perfect and we should resolve this in a different way)
```

Comment:
@jezdez I guess this is another reason to split the query runners into their own packages.

Reply:
Why #2603 is another case?

Reply:
Anyway, I think it's worth merging this regardless. In parallel we need to layout a plan to streamline our release process and part of that separate the query runners into their own packages.

Reply:
I mean, #2603 shows that some of the query runners have exceedingly complicated dependencies on their own for which separation from the Redash core is useful. My underlying hypothesis is of course that a lean Redash core is easier to maintain than a "full" Redash core.

Reply:
I see it the other way: this shows that we should make sure to always test with all the dependencies, as some query runners might introduce conflicting packages or just require a different version than we would otherwise use.

_But_ we can always run the full test suite with all the dependencies before releasing.

Reply:
I'm not arguing against running the Redash tests will all the query runners, but for separating the code of the query runners in separate packages. They would still be installed via the requirements_all_ds.txt (before release) but simply install redash-cassandra, redash-mongo etc instead.